src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -533,7 +536,7 @@ def IsConnected(self):
                 self.devCtrl)
         )
 
-    def ConnectBLE(self, discriminator, setupPinCode, nodeid) -> PyChipError:
+    def ConnectBLE(self, discriminator, setupPinCode, nodeid) -> None:
```

Comment:
Watch-out, this will conflict with https://github.com/project-chip/connectedhomeip/pull/33933

Reply:
Thanks for the heads up, will rebase!